pkg/scheduler/framework/plugins/interpodaffinity/interpod_affinity.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package interpodaffinity
+
+import (
+	"fmt"
+
+	"k8s.io/api/core/v1"
+	"k8s.io/kubernetes/pkg/scheduler/algorithm"
+	"k8s.io/kubernetes/pkg/scheduler/algorithm/predicates"
+	"k8s.io/kubernetes/pkg/scheduler/framework/plugins/migration"
+	framework "k8s.io/kubernetes/pkg/scheduler/framework/v1alpha1"
+	"k8s.io/kubernetes/pkg/scheduler/nodeinfo"
+)
+
+// InterPodAffinity is a plugin that checks inter pod affinity
+type InterPodAffinity struct {
+	predicate predicates.FitPredicate
+}
+
+var _ = framework.FilterPlugin(&InterPodAffinity{})
```

Comment:
This is more common

```
var _ framework.FilterPlugin = &InterPodAffinity{}
```

Reply:
I have no preference on this myself. Mainly to keep consistent with most of the existing plugins(including the one of imagelocality you implemented) :)

Reply:
done

Reply:
I have sent https://github.com/kubernetes/kubernetes/pull/84073 to update the rest